### DISCUSSION TITLE
ocelot怎么保证consul可用

### DISCUSSION THREAD

**Discussion by 18682788952** [ThreeMammals/Ocelot#1980] [Q&A]
比如我consul是一个集群，ocelot对应consul节点配置只有一个，如果这个节点挂掉了，那网关不也就挂掉了么，那我搭建consul集群有何意义，怎么保证consul节点挂掉之后，网关调用其他consul节点，来实现高可用？

**Comment by ysq5202121** [comment]
大兄第，我也有这个疑问，莫非ocelot不支持consul集群？

**Comment by ysq5202121** [comment]
@18682788952 看了下历史的issues，发现似乎需要自己做代理，然后分发.
可以是nginx,或者IIS给集群做负载均衡，然后配置到ocelot里面去

**Comment by ysq5202121** [comment]
有什么研究发现，请分享

**Comment by raman-m** [comment]
Sorry, guys, I don't understand Chinese.
Regarding Ocelot vs Consul cluster...
I'm open to any discussions, and I invite you to contribute!
